docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/studentinternetaccessdim-view-experimental.md

```diff
@@ -0,0 +1,76 @@
+# StudentInternetAccessDim View - Experimental
+
+:::warning
+
+This view is already in the `main` branch and will be included in the Analytics
+Middle Tier version 2.5.0. This is considered experimental functionality with
+its initial release and may be subject to significant changes during the course
+of ongoing work with
+the [https://edfi.atlassian.net/wiki/spaces/SKD/pages/26841349](https://edfi.atlassian.net/wiki/spaces/SKD/pages/26841349).
+Because of the experimental nature, patch releases of the Analytics Middle Tier
+might have _breaking changes_ in this view, so it is not recommended that anyone
+take a hard dependency on this view until further notice.
+
+:::
+
+## Purpose
+
+Describe's a Student's Internet Access, as defined for a student's relationship
+with the _school._
+
+## SQL Object
+
+`analytics.AcademicTimePeriodDim`
+
+## Usage Notes
+
+❌ Not available for Data Standard 2.2, as the underlying entities were
+added in Data Standard 3.
+
+⚠️ As with many other views, this query relies on the Student's
+relationship with an EducationOrganization, via
+the `StudentEducationOrganization`  table in the ODS. Thus it is possible that
+the Student's InternetAccess could be recorded for the student's relationship
+with one or more agency, including School, Local Education Agency, Service
+Network, etc. For this view's purpose, only the School relationship is
+considered. Data recorded for any other type of Education Agency will be
+ignored.
```

Comment:
Do we think these need any additional admonishments? Or perhaps the warnings can all be combined in one area?

Reply:
Good questions, but let's leave them be for now.

Reply:
Now that I'm making other changes, I'll update these to use an admonishment.